components/Button.js

```diff
@@ -0,0 +1,11 @@
+export default function Button({
```

Comment:
Try to use a name that defines more the characteristics of the button, maybe for the color.

Reply:
Isn't the component aimed to be a general purpose button? Rather than a specific one.

Reply:
I agree the naming could be improved to avoid confusions with HTML's buttons.

Maybe GenericButton? Not sure about this one.